core/services/postgres/q.go

```diff
@@ -61,7 +64,14 @@ func WithParentCtx(ctx context.Context) func(q *Q) {
 	}
 }
 
+func WithLogger(lggr logger.Logger) func(q *Q) {
+	return func(q *Q) {
+		q.lggr = lggr
+	}
+}
+
 var _ Queryer = Q{}
+var slowThreshold = time.Millisecond
```

Comment:
You probably want `time.Second` here

Reply:
Maybe even consider using an env var (that can be set to 0 to disable slow query logging)

Reply:
ah.. a debug leftover :)
Agreed, will add a new ENV variable